### PR TITLE
Various build process improvements

### PR DESCRIPTION
**Pull request by linearregression**
* Add travisci support
* run code coverage, post result to coveall.io, add coverall.io badges
* add project info for hex package

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,5 +10,11 @@ before_install:
   - wget http://www.apache.org/dist/thrift/0.9.3/thrift-0.9.3.tar.gz
   - tar xfz thrift-0.9.3.tar.gz
   - cd thrift-0.9.3 && ./configure --with-erlang --without-cpp --without-csharp --without-java --without-nodejs --without-lua --without-python --without-perl --without-php --without-php_extension --without-ruby --without-haskell --without-go --without-haxe --without-d && sudo make install && cd ..
+  - mix local.hex --force
```
Comment (scohen) [inline .travis.yml]: @jparise mind reviewing the travis stuff?

File: mix.exs [hunk 2]
```diff
@@ -46,7 +48,27 @@ defmodule Riffed.Mixfile do
...
+
+  defp project_url do
+    """
+    https://github.com/pinterest/riffed
+    """
+  end
+
+  defp package do
+    [files: ["config", "lib", "test", "thrift", "mix.exs", "README.md", "LICENSE"],
+     maintainers: ["Jon Parise", "Steve Cohen"],
```
Comment (scohen) [inline mix.exs]: Ouch... Always a bridesmaid.

**Comment by scohen** [comment]
This LGTM.

**Comment by jparise** [comment]
This looks good. I've also just enabled Coveralls support for this repository, so we should be good to go.

**Pull request merged**